CHANGELOG.md

```diff
@@ -1,6 +1,277 @@
 Changelog
 =========
 
+0.5.0
+-----
+
+**Date:** 2020-03-XX
+
+Perfectly Matched Layer (PML) and Bug Fixes
+
+This release introduce PML boundary conditions for the Yee field solver and particle 
+rendering with the ISAAC plugin.
+
+Thanks to Igor Andriyash, Sergei Bastrakov, Xeinia Bastrakova, Andrei Berceanu,
+Finn-Ole Carstens, Alexander Debus, Jian Fuh, Marco Garten, Axel Huebl,
+Sophie Kossag, Anton Lebedev, Felix Meyer, Pawel Ordyna, Richard Pausch,
```

Comment:
`Koßagk` but now she's `Sophie Rudat`. How do we want to handle new names?
Just update them or write `Sophie Rudat (Koßagk)`?

Reply:
I will keep Koßagk because here github account is showing this name.